src/MongoDB/ReadConcern.c

```diff
@@ -197,6 +203,102 @@ static PHP_METHOD(ReadConcern, bsonSerialize)
 	convert_to_object(return_value);
 } /* }}} */
 
+/* {{{ proto string MongoDB\Driver\ReadConcern::serialize()
+*/
+static PHP_METHOD(ReadConcern, serialize)
+{
+	php_phongo_readconcern_t* intern;
+	ZVAL_RETVAL_TYPE          retval;
+	php_serialize_data_t      var_hash;
+	smart_str                 buf = { 0 };
+	const char*               level;
+
+	intern = Z_READCONCERN_OBJ_P(getThis());
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	if (!intern->read_concern) {
+		return;
+	}
+
+	level = mongoc_read_concern_get_level(intern->read_concern);
+
+	if (!level) {
+		PHONGO_RETURN_STRING("");
```

Comment:
I assume this was inspired by the MinKey and MaxKey methods? AFAIK, those are the only BSON types with no internal data.

Reply:
That is correct.